randovania/games/am2r/json_data/Industrial Complex.txt

```diff
@@ -1362,7 +1372,7 @@ Extra - map_name: rm_a3b05
   * Layers: default
   * Open Passage to Breeding Grounds Gamma Nest Middle/Dock to Breeding Grounds Alpha Nest Access
   > Dock to Breeding Grounds Alpha Nest
-      Space Jump or Walljump (Advanced) or Can Use Spider Ball
+      Hi-Jump Boots or Space Jump or Walljump (Advanced) or Can Use Spider Ball
```

Comment:
i dont think you can go through the room transition with just higjump. seems like hijump+beginner WJ to me at first glance